lib/command.js

```diff
@@ -1441,11 +1441,11 @@ Expecting one of '${allowedValues.join("', '")}'`);
   }
 
   /**
-   * Return an object containing options as key-value pairs
+   * Return an object containing local option values as key-value pairs.
    *
    * @return {Object}
    */
-  opts() {
+  opts(mergeOptions) {
```

Comment:
Is the `mergeOptions` argument unnecessary?

Reply:
Oops, I think that is left over from first try. Thanks.

Reply:
Removed. And the copy-and-paste copy of it too. 😊